src/components/metadata/metadata.js

```diff
@@ -36,7 +36,7 @@ const MetaData = ({
   // Hide plot modal when selected metadata changes
   useEffect(() => onTogglePlotModal(false), [metadata, onTogglePlotModal]);
   const isTaskNode = metadata?.type === 'task';
```

Comment:
Maybe add a comment after this line to say that this really should be `isNodeNode`, though this `isTaskNode` naming is better, hence we're leaving it, as you said in your PR description?

Reply:
I'd actually leave the `isDataNode` as it was called rather than renaming to `isDatasetNode`. That way everything is consistent and `isTaskNode` would be "correct" already. The `isXNode` has `X` consistent with the backend naming scheme. This way the only inconsistency is the way the names appear on the screen, which is all handled by `translateMetadataType`.

Reply:
yea @AntonyMilneQB I agree on this, and was think the same thing, so you kind of confirmed my thoughts!

I'll just revert it back to `isDataNode`.